example_configs/text2speech/wavenet_float.py

```diff
@@ -0,0 +1,104 @@
+# pylint: skip-file
+import tensorflow as tf
+from open_seq2seq.models import Text2SpeechWavenet
+from open_seq2seq.encoders import WavenetEncoder
+from open_seq2seq.decoders import FakeDecoder
+from open_seq2seq.losses import WavenetLoss
+from open_seq2seq.data import WavenetDataLayer
+from open_seq2seq.optimizers.lr_policies import exp_decay
+from open_seq2seq.parts.convs2s.utils import gated_linear_units
+
+base_model = Text2SpeechWavenet
+
+base_params = {
+  "random_seed": 0,
+  "use_horovod": True,
```

Comment:
I would recommend to disable horovod for less than 4 gpus unless you notice a speed difference